### PR TITLE
Fix build scripts

### PR DESCRIPTION
**Pull request by KTibow**
This fixes the build scripts so they work.

### CODE REVIEW

**Comment by nielsfaber** [comment]
For me they work as they are now.
I don’t even have `npx` installed AFAIK.
Are you sure this is an improvement?

**Comment by KTibow** [comment]
What do you mean? I didn't know it was optional. I'll remove that, and it'll fix the button-card part.

File: package.json [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
   "scripts": {
     "build": "npm run lint && npm run rollup && npm run babel",
-    "rollup": "rollup -c",
-    "babel": "babel dist/button-card.js --out-file dist/button-card.js",
-    "lint": "eslint src/**/*.ts --fix",
-    "start": "rollup -c --watch"
+    "rollup": "npx rollup -c",
+    "babel": "npx babel dist/scheduler-card.js --out-file dist/scheduler-card.js",
+    "lint": "npx eslint src/**/*.ts --fix",
+    "start": "npx rollup -c --watch"
```
Comment (KTibow) [inline package.json:11]:
```suggestion
    "rollup": "rollup -c",
    "babel": "babel dist/scheduler-card.js --out-file dist/scheduler-card.js",
    "lint": "eslint src/**/*.ts --fix",
    "start": "rollup -c --watch"
```

**Comment by nielsfaber** [comment]
I don’t know how these things are supposed to work.
For me `npm start` does the trick.
Not sure what the difference with `npx` is.

Thanks for taking the effort to improve things anyway!

**Pull request merged**